Library/Homebrew/language/node.rb

```diff
@@ -1,10 +1,44 @@
+require "json"
+
 module Language
   module Node
     def self.npm_cache_config
       "cache=#{HOMEBREW_CACHE}/npm_cache"
     end
 
-    def self.pack_for_installation
+    def self.node_system(cmd, *args)
```

Comment:
How to do so? Using just `system` inside a `language` will call [`Homebrew/utils.rb`system](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/utils.rb#L185) and not [`Homebrew/formula.rb` one](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/formula.rb#L1720).

An alternative implementation of this using util's `_system`(basically mixing the implementation of `safe_system` and `quite_system) would be:
```ruby
def self.node_system(cmd, *args)
  ohai "#{cmd} #{args * " "}"
  Homebrew._system(cmd, *args) do
    unless ARGV.verbose?
      $stdout.reopen("/dev/null")
      $stderr.reopen("/dev/null")
    end
  end
  raise ErrorDuringExecution.new(cmd, args) unless $CHILD_STATUS.success?
end

``